harness/tests/experiment/keras/test_tf_keras_trial.py

```diff
@@ -593,3 +593,33 @@ def make_workloads_with_metrics(metrics_storage: List) -> workload.Stream:
 
     assert len(trial_B_metrics) == len(trial_C_metrics) == 5
     assert trial_B_metrics == trial_C_metrics
+
+
+@pytest.mark.tensorflow
+def test_keras_from_config() -> None:
+    from tests.experiment.fixtures import tf_keras_one_var_model
```

Comment:
I suggest moving this to the top of the file or adding a comment about why it is here.